static/src/javascripts/projects/common/modules/commercial/acquisitions-ophan.js

```diff
@@ -39,22 +39,27 @@ export const submitViewEvent = (componentEvent: ComponentEventWithoutAction) =>
         action: 'VIEW',
     });
 
+export const addReferrerData = (acquisitionData: {}): {} =>
+    // Note: the current page is the referrer data in the context of the acquisition.
+    Object.assign({}, acquisitionData, {
```

Comment:
I steered you wrong earlier with this one Guy - I forgot we had object spread syntax in frontend now, which is probably the most concise way:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator

```
{
  ...acquisitionData,
  referrerPageviewId: config.get('ophan.pageViewId'),
  referrerUrl: window.location.href.split('?')[0]
}
```